cmd/gazelle/fix_test.go

```diff
@@ -112,12 +112,15 @@ func TestCreateFile(t *testing.T) {
 	defer os.RemoveAll(dir)
 
 	goFile := filepath.Join(dir, "main.go")
-	if err = ioutil.WriteFile(goFile, []byte("package main"), 0600); err != nil {
+	if err = ioutil.WriteFile(goFile, []byte("package main"), 0o600); err != nil {
 		t.Fatalf("error writing file %q: %v", goFile, err)
 	}
 
 	// Check that Gazelle creates a new file named "BUILD.bazel".
-	run(dir, defaultArgs(dir))
+	err = run(dir, defaultArgs(dir))
+	if err != nil {
+		t.Fatalf("run failed: %v", err)
+	}
```

Comment:
```suggestion
	if err := run(dir, defaultArgs(dir)); err != nil {
		t.Fatalf("run failed: %v", err)
	}
```

Try to reduce the scope of `err`: https://github.com/uber-go/guide/blob/master/style.md#reduce-scope-of-variables

Can you use this pattern to fix other err_check?

Reply:
Thanks, I applied the suggested style on top of the errcheck commit.

It would be nice if Uber could provide an analysis.Analyzer that check for these cases. I would love to make it a part of nogo analysis